mne/stats/cluster_level.py

```diff
@@ -500,7 +501,7 @@ def _find_clusters_1dir(x, x_in, connectivity, max_step, t_power, ndimage):
         if x.ndim == 1:
             # slices
             clusters = ndimage.find_objects(labels, n_labels)
-            if len(clusters) == 0:
+            if not clusters:
```

Comment:
I find this less clear actually because before it was clear we were considering if there were any elements in an iterable (len-0), with the current code it's not clear if it's checking for boolean / None / len-0, etc.

Reply:
I originally thought that because `_find_clusters_1dir` is called in a loop (of another loop...) and `clusters` is always a list of None or slices, `not clusters` would be better to save some time. I would run some more tests to check.

Reply:
`if not clusters` cost 14.7904 s while `if len(clusters) == 0` cost 14.9306 s when n_perm=1000, n_space=1000, n_time=1000. `not clusters` could bring a 0.93% speedup (I found this [answer](https://stackoverflow.com/a/45778282) helpful). Running permutation test on my dataset cost about 7 hours, that would save about 4 minutes. Not much time saved indeed...Not sure what to do, what do you think?

Reply:
That's more than I expected actually. Maybe just add a comment that it's equiv to checking len(x) == 0 but faster